### ISSUE TITLE
Negative loss while pretraining

### ISSUE THREAD

**Issue by charlesxu90** [junxia97/Mole-BERT#21]
Dear @junxia97 ,

I'm trying to replicate the experiments by retrain the vq-vae and CL model. The VQ-VAE model runs smoothly, however, when training the CL model, I found the negative loss during training.

Through I saw the `acc_node` and `acc_edge` is increasing. However, it seems weired to have a negative loss during training.
```shell
epoch 1 iter 9122: loss -3.14923,  acc_node 0.70855,  acc_edge 0.97823.:  15%|██████████████████████████████████████▋                                                                                                                                                                                                                                  | 9123/62500 [06:35<38:56, 22.85it/s]
```

Any recommendations for such issue?

BTW, is it possible that the checkpoints of the VQ-VAE models to be shared? Such as we can finetune it on some other datasets?

Best regards,

**Comment by charlesxu90** [comment]
BTW, any specific parameter setups for the code? I'm not sure if that's a problem with my implementation. But in pretraining, the loss after the 1st epoch becomes `nan`.

**Comment by charlesxu90** [comment]
I test the checkpoint after the first epoch on `sider` dataset. It can achieve 0.7 auc, which is unexpected.

The resulting code are availe [here](https://github.com/charlesxu90/gps_demo/tree/main/mole-bert).

I mainly simpify the code for me to understand what is going on.

**Comment by cc421528** [comment]
Hello, I have encountered both of the issues you mentioned now. Have you resolved these issues now? If so, could you please share your thoughts with me. Thank you very much!!

**Comment by charlesxu90** [comment]
It turns out to be some manipulation of the input. the input is [-1, 1], so the output should be adjusted into the same range.
